src/main/java/org/mule/extension/api/annotation/param/display/Placement.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.extension.api.annotation.param.display;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Defines the placement of a {@link org.mule.extension.annotation.api.Parameter} field
```

Comment:
The fact that it defines the placement is kinda obvious. It'd be better if it explains what the placement means